package.json

```diff
@@ -118,8 +118,9 @@
     "vite-plugin-node-polyfills": "^0.22.0"
   },
   "dependencies": {
-    "@permaweb/aoconnect": "^0.0.55",
+    "@permaweb/aoconnect": "^0.0.57",
     "arbundles": "0.11.0",
+    "arconnect": "^1.0.3",
```

Comment:
is this being used? if not required, remove from the build or put as a dev dep

Reply:
This is the types package for arconnect, we do use it - will try as a dev dep

Reply:
https://github.com/ar-io/ar-io-sdk/pull/175/commits/34f07d25024c080f529218569cba57041e34baa2

Reply:
should be fine as a dev dep if we are not explicitly exporting any arconnect types from our own build